SECURITY.md

```diff
@@ -18,6 +18,20 @@ As Git for Windows bundles more than just Git (such as Bash, OpenSSL, OpenSSH, G
 
 Every Git for Windows version is tagged using a name that starts with the Git version on which it is based, with the suffix `.windows.<patchlevel>` appended. For example, Git for Windows v2.17.1' source code is tagged as [`v2.17.1.windows.1`](https://github.com/git-for-windows/git/releases/tag/v2.17.1.windows.1) (the patch level is always at least 1, given that Git for Windows always has patches on top of Git). Likewise, Git for Windows v2.17.1(2)' source code is tagged as [`v2.17.1.windows.2`](https://github.com/git-for-windows/git/releases/tag/v2.17.1.windows.2).
 
+## Release Candidate (rc) versions
+
+Git for Windows also releases versions that reflect the [upstream release candidates](https://tinyurl.com/gitCal). These contain the `-rc<n>` suffix to the expected regular git version, and before the 'windows' suffix. These releases are independent of upstream but are tied together by convention. It should be noted that these rc versions currently sort after their formal release, so appear to be newer to the updater software.
```

Comment:
The `.windows.<patchlevel>` suffix only applies to the tag names, not to the Git for Windows version numbers...

Reply:
> The `.windows.<patchlevel>` suffix only applies to the tag names, not to the Git for Windows version numbers...

Not sure what you mean here, so can't begin to change anything. Do you want to suggest an alternate?

Reply:
I was referring to this sentence:

> These contain the `-rc<n>` suffix to the expected regular git version, and before the 'windows' suffix.

It makes it sound as if the `.windows.` infix is part of the version number, but it is not.

Reply:
I have it (the `.<rc<n>`) as being between regular (upstream/linux) git (e.g. `2.22.0` and the `.windows.`, so it is as I said.  How would you phrase it?

Reply:
But it is not between the regular version and `.windows.`, because `.windows.` is not even part of the version number.

Talking about `.windows.` here is outright confusing.

Reply:
I say again, How would you phrase it?

$ git version
git version 2.22.0.rc3.windows.1.380.gef5ddde56c.dirty

Reply:
Users don't think about the Git for Windows version in terms of the output of `git version`, but in terms of the version provided on https://gitforwindows.org/, https://git-scm.com and https://github.com/git-for-windows/git/releases. There are no `.windows.` infixes there, nor funny `.dirty` suffixes. Ever.

So this is how I would have phrased it:

> Git for Windows also publishes versions based on Git's release candidates (for upcoming "`.0`" versions, see [Git's release schedule](https://tinyurl.com/gitCal)). These versions end in `-rc<n>`, starting with `-rc0` for a very early preview of what is to come, and as with regular versions, Git for Windows tries to follow Git's releases as quickly as possible.
>
> Note: there is currently a bug in the "Check daily for updates" code, where it mistakes the final version as a downgrade from release candidates. Example: if you installed Git for Windows v2.23.0-rc3 and enabled the auto-updater, it would ask you whether you want to "downgrade" to v2.23.0 when that version was available.

Of course, I had hoped that I would not need to spend time on phrasing this, I am not a native speaker after all ;-)